### PR TITLE
WIP: Unit tests for non-multiturn servos

### PR DESCRIPTION
**Pull request by hello-binit**
@aedsinger Wrote a unit test called "test_non_multiturn_move_after_home" that replicates the issue described in #12. Still seeing the wrong behavior, where the servo doesn't respond to move_to(0) at the end of homing. Could you take a look at this?

### CODE REVIEW

**Comment by aedsinger** [comment]
Ok. I had replicated the issue and resolved part of it at least. Will take a look again.

**Comment by aedsinger** [comment]
The issue is subtle. I've added documentation to the servo homing code. In single turn mode the zero and range values being read from YAML are in 'multi-turn units'.

I've updated the unit test to home the head-tilt. It should work now.

File: body/stretch_body/dynamixel_hello_XL430.py [hunk 1]
```diff
@@ -232,6 +232,7 @@ def enable_pos(self):
             self.motor.enable_multiturn()
         else:
             self.motor.enable_pos()
+            sf=self.motor.get_pos()
```
Comment (hello-binit) [inline body/stretch_body/dynamixel_hello_XL430.py:235]: @aedsinger Can I get rid of `sf` here?

**Comment by hello-binit** [comment]
The second unit test fails still fails on wrist_yaw (wrist_yaw doesn't react to move_to(0) as expected). Is this because the zero and range_t for wrist_yaw are calibrated for multi-turn?

File: body/stretch_body/dynamixel_hello_XL430.py (hunk 1 above)
Comment (aedsinger) [reply body/stretch_body/dynamixel_hello_XL430.py:235]: Ah yes, that's left over

**Comment by aedsinger** [comment]
Hmm, the code I committed dropped the wrist yaw test for a head tilt test. This is because the wrist yaw calibration is multi-turn. 

See: https://github.com/hello-robot/stretch_body/blob/bugfix/enable_pos_tests/body/test/test_dynamixel_hello_XL430.py

**Pull request merged**